### PR TITLE
Create bottom navbar

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:**  In the old design didn't exist the navbar.

**Solution:** Create navbar according to the new design.

**TODO/FIXME:** Add animation to changes between display none and display block.

final

### CODE REVIEW

File: frontend/main/mainController.js [hunk 1]
```diff
@@ -173,14 +175,33 @@
             $window.location.reload();
         };
 
+        mainCtrl.goToNavBottom = function goToNavBottom(state) {
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: go to nav bottom?

File: frontend/main/mainController.js [hunk 2]
```diff
@@ -173,14 +175,33 @@
             $window.location.reload();
         };
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: js doc for all functions

File: frontend/main/mainController.js [hunk 3]
```diff
@@ -173,14 +175,33 @@
...
 
+        mainCtrl.goToNavBottom = function goToNavBottom(state) {
+            mainCtrl.stateView = state;
+            document.getElementById('main-toolbar').style.display = 'block';
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: make a function `resetNavBarDisplayStyle`

File: frontend/main/mainController.js [hunk 4]
```diff
@@ -173,14 +175,33 @@
...
+        }
+
+        mainCtrl.getSelectedItemClass = function getSelectedItemClass(state){
+            loadStateView();
+            return (state === mainCtrl.stateView) ? "color-icon-selected-navbar":"color-icon-navbar";
+        };
+
+        function loadStateView(){
+            mainCtrl.stateView = $state.current.name.split(".")[2];
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: use full state name.

File: frontend/main/mainController.js [hunk 5]
```diff
@@ -173,14 +175,33 @@
             $window.location.reload();
         };
 
+        mainCtrl.goToNavBottom = function goToNavBottom(state) {
+            mainCtrl.stateView = state;
+            document.getElementById('main-toolbar').style.display = 'block';
+            $state.go('app.user.' + state);
+        };
+
+        mainCtrl.goToNotifications = function goToNotifications(){
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: its not going anywhere.

File: frontend/sw.js [hunk 6]
```diff
@@ -7,7 +7,7 @@
     importScripts('app/firebase-config.js');
     importScripts('app/config.js');
     // if the line number of the code below changes, modify the /ecis script.
-    const CACHE_SUFIX = 'master';
+    const CACHE_SUFIX = 'bottom-menu';
```
Comment (JuliePessoa) [inline frontend/sw.js]: undo

File: frontend/user/left_nav.html [hunk 7]
```diff
@@ -48,13 +48,6 @@
   
               <div class="row section-menu">
                 <md-menu-content width="4" md-colors="{background: 'grey-100'}">
-                  <md-menu-item>
```
Comment (JuliePessoa) [inline frontend/user/left_nav.html]: this only should be removed on mobile screen

File: frontend/user/left_nav.html [hunk 8]
```diff
@@ -86,21 +79,8 @@
                 <md-menu-content width="4" md-colors="{background: 'grey-100'}">
                   <md-menu-item>
                     <md-button ng-click="mainCtrl.goToEditInfo()">
-                      <md-icon>edit</md-icon>
-                      <b>Editar Informações</b>
```
Comment (JuliePessoa) [inline frontend/user/left_nav.html]: `Editar Informações` should be keeped. in the same place as `Editar Perfil`

File: frontend/user/left_nav.html [hunk 9]
```diff
@@ -86,21 +79,8 @@
...
-                        data-badge>account_circle</md-icon>
-                      <b>Gerenciar Membros</b>
-                  </md-menu-item>
-                  <md-menu-item>
-                    <md-button ng-click="mainCtrl.goToManageInstitutions()">
-                      <md-icon ng-class="mainCtrl.pendingInstLinksInvitations? 'notification-badge': ''"
-                        data-badge>account_balance</md-icon>
-                      <b>Vínculos Institucionais</b>
+                      <md-icon>account_balance</md-icon>
+                      <b>Gerenciar instituição</b>
```
Comment (JuliePessoa) [inline frontend/user/left_nav.html]: Its is going to Edit info on click. is it right?

File: frontend/utils/utils.js [hunk 10]
```diff
@@ -200,6 +200,14 @@ var Utils = {
         return screen.width <= 960;
     },
 
+    /**
+     * This function indicate if the screen are in phone device size.
+     * @returns {boolean} True if the screen is smaller or equal to 450 pixels and false in otherwise.
+     */
+    isPhoneScreen: function isPhoneScreen() {
```
Comment (JuliePessoa) [inline frontend/utils/utils.js]: test

File: frontend/main/mainController.js [hunk 11]
```diff
@@ -173,14 +175,33 @@
...
+            mainCtrl.stateView = "notifications";
+            document.getElementById('main-toolbar').style.display = 'block';
+        }
+
+        mainCtrl.getSelectedItemClass = function getSelectedItemClass(state){
```
Comment (JuliePessoa) [inline frontend/main/mainController.js]: test

File: frontend/main/main.html [hunk 12]
```diff
@@ -67,4 +67,33 @@ <h2 class="md-flex">Verifique seu email para confirmar sua conta.</h2>
...
+    <div class="icon-navbar-bottom">
+      <md-button class="md-icon-button" aria-label="Eventos" ng-click="mainCtrl.goToOfBottomNav('events')">
+        <md-icon ng-class="mainCtrl.getSelectedStateClass('events')">event</md-icon>
+      </md-button>
+      <p ng-class="mainCtrl.getSelectedStateClass('events')" id="label-icon-navbottom">Eventos</p>
+    </div>
+    <div class="icon-navbar-bottom">
+    <md-button class="md-icon-button big-label-icon" aria-label="Instituições" ng-click="mainCtrl.goToOfBottomNav('institutions')">
+        <md-icon ng-class="mainCtrl.getSelectedStateClass('institutions')">account_balance</md-icon>
+      </md-button>
```
Comment (raonismaneoto) [inline frontend/main/main.html]: fix the identation

File: frontend/main/mainController.js [hunk 13]
```diff
@@ -173,14 +175,46 @@
...
+        mainCtrl.selectNotificationState = function selectNotificationState(){
+            mainCtrl.stateView = "app.user.notifications";
+            resetNavBarDisplayStyle();
+        }
+
+        /** Return correct class according currently state view.
+         */
+        mainCtrl.getSelectedStateClass = function getSelectedStateClass(state){
+            state = "app.user." + state;
+            console.log(state, state === mainCtrl.stateView);
```
Comment (raonismaneoto) [inline frontend/main/mainController.js]: remove it

File: frontend/styles/custom.css [hunk 14]
```diff
@@ -339,6 +339,31 @@ a:active {
...
+      opacity: 1;
+    }
+}
+
+@keyframes fadeOut {
+    0% {
+        opacity: 1;
+    }
+
+    60% {
```
Comment (JuliePessoa) [inline frontend/styles/custom.css]: both 60?

File: frontend/sw.js (hunk 6 above)
Comment (JuliePessoa) [reply frontend/sw.js]: undo

File: frontend/utils/utils.js (hunk 10 above)
Comment (raonismaneoto) [inline frontend/utils/utils.js]: your branch came from dev or responsive? If it came from responsive merge with it and then use isMobileScreen function, there this function receive the screenSize as parameter. If it came from dev, it's ok for me.

**Pull request merged**